mono/Makefile.am

```diff
@@ -6,11 +6,15 @@ if BTLS
 btls_dirs = btls
 endif
 
+if PAL_SUPPORTED
+pal_dirs = pal
```

Comment:
Could we name this so that it's clear that the PAL in question is not Mono's to prevent misuse ? E.g. corefx_pal